src/decision/location_selection.jl

```diff
@@ -0,0 +1,356 @@
+
+# Arguments
+- `ranks` : Contains location ranks for each scenario of location selection, as created by 
+    `run_location_selection()`.
+- `rs` : ADRIA result set.
+- `ind_metrics` : Indices for selected scenarios (such as robust scenarios).
+- `iv_type` : indicates intervention log to use ("seed", "shade" or "fog").
+- `n_loc_int` : number of locations which are intervened at for each intervention decision.
+
+# Returns 
+- Counts for location selection at each location in the domain.
+"""
+function location_selection_frequencies(
+    ranks::NamedDimsArray,
+    iv_type::String;
+    n_loc_int::Int64=5,
+    ind_metrics::Vector{Int64}=_get_scen_ids(ranks),
+)
+    ranks_frequencies = ranks_to_frequencies(ranks[scenarios=ind_metrics], iv_type; n_ranks=n_loc_int)
+    loc_count = sum(ranks_frequencies[ranks=1:n_loc_int], dims=2)[ranks=1]
+
+    return loc_count
+end
+function location_selection_frequencies(
+    rs::ResultSet,
+    iv_type::String;
+    n_loc_int::Int64=5,
+    ind_metrics::Vector{Int64}=_get_scen_ids(rs.ranks),
+)
+    selected_ranks = _get_iv_type(rs.ranks[scenarios=ind_metrics], iv_type)
+    ranks_frequencies = ranks_to_frequencies_ts(selected_ranks; n_ranks=n_loc_int)
+    loc_count = dropdims(sum(ranks_frequencies[ranks=1:n_loc_int], dims=[1, 3]), dims=3)[timesteps=1]
+
+    return loc_count
+end
+
+"""
+    summed_inverse_rank(ranks::NamedDimsArray, iv_type::String; dims=:scenarios,agg_func=x->x)
+    summed_inverse_rank(rs::ResultSet, iv_type::String; dims=:scenarios,agg_func=x->x)
+    summed_inverse_rank(ranks::NamedDimsArray; dims=:scenarios,agg_func=x->x)
+ 
+Calculates (number of sites) .- ranks summed over the dimension dims and transformed using agg_func 
+    (default no transformation).
+
+# Arguments
+- `ranks` : Contains location ranks for each scenario of location selection, as created by 
+    `run_location_selection()`.
+- `rs` : ADRIA result set.
+- `iv_type` : indicates intervention log to use ("seed", "shade" or "fog").
+- `dims` : Dimensions to sum over.
+- `agg_func` : function to transform result.
+
+# Returns 
+- Inverse rankings (i.e. the greater the number the higher ranked the site).
+"""
+function summed_inverse_rank(
+    ranks::NamedDimsArray,
+    iv_type::String;
+    dims::Symbol=:scenarios,
+    agg_func::Function=x -> x,
+)
+    selected_ranks = _get_iv_type(ranks, iv_type)
+    return summed_inverse_rank(selected_ranks; dims=dims,agg_func=agg_func)
+
+end
+function summed_inverse_rank(
+    rs::ResultSet, iv_type::String; dims::Symbol=:scenarios, agg_func::Function=x -> x
+)
+    selected_ranks = _get_iv_type(rs.ranks, iv_type)
+    return summed_inverse_rank(selected_ranks; dims=dims,agg_func=agg_func)
+
+end
+function summed_inverse_rank(
+    ranks::NamedDimsArray; dims::Symbol=:scenarios, agg_func::Function=x -> x
+)
+    n_ranks = maximum(ranks)
+    inv_ranks = agg_func(dropdims(sum(n_ranks .- ranks,dims=dims),dims=dims))
```

Comment:
Fix up spacing between arguments please.

(I'm surprised the formatter isn't fixing this for you)